### PR TITLE
CSPM-1822: asset counter

### PR DESCRIPTION
**Pull request by senalplerion**
**Jira Link:**
[CSPM-1822](https://plerion.atlassian.net/browse/CSPM-1822)

**Summary/Motivation:**

- PLERION-AWS-175
  - As a Plerion Platform user I want to receive a ‘PASSED' finding if Amazon Macie is enabled
  - As a Plerion Platform user I want to receive a ‘FAILED' finding if Amazon Macie is disabled
- PLERION-AWS-811
  - As a Plerion Platform user I want to receive a ‘PASSED’ finding if the AWS account associated to my integration is part of an AWS Organization
  - As a Plerion Platform user I want to receive a ‘FAILED’ finding if the AWS account associated to my integration is not part of an AWS Organization
  - As a Plerion Platform user I want to receive an ‘UNKNOWN' finding if the AWS account associated to my integration is the Organization’s managed account
- PLERION-AWS-812
  - As a Plerion Platform user I want to receive a ‘PASSED' finding if Amazon Inspector is enabled
  - As a Plerion Platform user I want to receive a ‘FAILED' finding if Amazon Inspector is disabled

[CSPM-1822]: https://plerion.atlassian.net/browse/CSPM-1822?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: AWS/aws-dependencies.json [hunk 1]
```diff
@@ -1357,5 +1357,66 @@
...
+          }
+        },
+        "listClientConfig": {
+          "clientName": "Inspector2Client",
+          "clientType": "AWS_V3",
+          "packageName": "@aws-sdk/client-inspector2"
+        }
+      }
+    ]
+  },
```
Comment (plerionedmund) [inline AWS/aws-dependencies.json]: duplicate of below
```suggestion
```

Comment (senalplerion) [reply AWS/aws-dependencies.json]: removed duplicate configs

File: AWS/aws-services.json [hunk 2]
```diff
@@ -2279,5 +2279,32 @@
       "me-south-1",
       "us-east-2"
     ]
+  },
+  {
+    "service": "Macie",
```
Comment (plerionedmund) [inline AWS/aws-services.json:2292]: ❓ do you need to add one for organization account and inspectorv2 account?

Comment (senalplerion) [reply AWS/aws-services.json:2292]: Added configs for inspector and organization account

**Review by plerionedmund — APPROVED** [review]

**Pull request merged**